src/app/types/table.ts

```diff
@@ -1,5 +1,8 @@
+import type { JSX } from "react";
+
 export interface Column<T> {
   header: string | React.ReactNode;
   accessor?: keyof T;
   render?: (item: T) => React.ReactNode;
+  Cell?: (row: { original: T }) => JSX.Element;
```

Comment:
Esto debería estar en minúscula

```suggestion
  cell?: (row: { original: T }) => JSX.Element;
```